MrAaronOlsen/trail-up
# Default trail image

#### What does  this PR do?
Adds JS function to check for image and add default if not

#### Where should the reviewer start?
simple_map.js

#### How should this be manually tested?
Run rails server. Go to trail search. Check that map point has image associated with trail on default. If you search 07078 the lower right non-clustered map point should have the default image(logo) instead.

#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
  - Do Migrations Need to be ran? 
no
  - Do Environment Variables need to be set? 
no
  - Any other deploy steps? 
no
